src/crud.js

```diff
@@ -71,13 +71,6 @@ export default (server, model, { prefix, defaultConfig: config, models: permissi
     return params;
   }, {});
 
-  const validAssociations = modelAssociations.length
```

Comment:
Why move this validation into `get-config-for-method`? All other validation is defined in this method already.